src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.ts

```diff
@@ -111,6 +118,45 @@ export class HearingActualAddEditSummaryComponent implements OnInit, OnDestroy {
     return hearingTypeFromLookup ? hearingTypeFromLookup.value_en : '';
   }
 
+  private getActualHearingDay(hearingActualsMainModel: HearingActualsMainModel) {
+    return hearingActualsMainModel.hearingActuals.actualHearingDays && hearingActualsMainModel.hearingActuals.actualHearingDays.length > 0
+      ? hearingActualsMainModel.hearingActuals.actualHearingDays[0] : null;
+  }
+
+  private getActualDayParties(hearingActualsMainModel: HearingActualsMainModel): void {
+    if (HearingActualAddEditSummaryComponent.hasActualParties(hearingActualsMainModel, this.hearingRoles)) {
+      const parties = hearingActualsMainModel.hearingActuals.actualHearingDays[0].actualDayParties;
+      for (const party of parties) {
+        if (party.partyRole === 'appellant' || party.partyRole === 'claimant') {
+          this.participants.push(party);
```

Comment:
A bit misleading, is this this.actualDayParties rather than this.participants? As we know participants have no appellant or claimant role, should we revert the names in the html show the this.actualDayParties first then this.participants.